jupyter-notebooks/analysis-ready-data/ard_1_intro_and_best_practices.ipynb

```diff
@@ -428,23 +596,33 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 26,
    "metadata": {
     "scrolled": false
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{\"_links\": {\"_self\": \"https://api.planet.com/compute/ops/orders/v2/eb65f433-61fd-4835-a2ad-2d6d007494f0\"}, \"created_on\": \"2023-06-22T11:01:14.824Z\", \"error_hints\": [], \"id\": \"eb65f433-61fd-4835-a2ad-2d6d007494f0\", \"last_message\": \"Preparing order\", \"last_modified\": \"2023-06-22T11:01:14.824Z\", \"name\": \"test_order_sdk_method_2\", \"products\": [{\"item_ids\": [\"20200111_145817_0f1a\", \"20200111_145816_1_0f1a\"], \"item_type\": \"PSScene\", \"product_bundle\": \"analytic_sr_udm2\"}], \"state\": \"queued\", \"tools\": [{\"clip\": {\"aoi\": {\"coordinates\": [[[-93.299129, 42.699599], [-93.299674, 42.812757], [-93.288436, 42.861921], [-93.265332, 42.924817], [-92.993873, 42.925124], [-92.993888, 42.773637], [-92.998396, 42.754529], [-93.019154, 42.699988], [-93.299129, 42.699599]]], \"type\": \"Polygon\"}}}, {\"bandmath\": {\"b1\": \"(b4 - b3) / (b4+b3)\", \"pixel_type\": \"32R\"}}]}\n"
+     ]
+    }
+   ],
    "source": [
     "!planet orders create 'json_request.txt'"
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "note that we see our Order ID above."
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
```

Comment:
In the paragraph below we're referencing tickets which are >3yr old at this point and completely irrelevant. Do we still feel like they're worth keeping @mkshah605? In my opinion, I would say no and I would suggest removing the first paragraph, entirely, and only mention we're downloading the data using the SDK using the Orders API, then mention the manifest file.

Reply:
@kevinlacaille would you be able to specify which section/tickets you mean here? I'm not exactly sure which paragraph you're referencing.

Reply:
Sorry, I couldn't comment, as it wasn't a new addition. But I as referring to the text:

"To download the order from the orders API, we will use the planet python client CLI. It would be nice to use the python client python API for this step but, as of the writing of this tutorial, support for the orders API in the planet client python API has been [confusing](https://github.com/planetlabs/planet-client-python/issues/217). The CLI download status output is also [confusing and possibly wrong](https://github.com/planetlabs/planet-client-python/issues/218) but the CLI does easily and successfully download the order."

Reply:
Yes, I agree, this ticket is now closed and we've since completely changed our documentation for the Client/CLI. I think we're safe to remove these references and simplify this for the reader.